internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/integration/utils/PubSubBrokerWrapper.java

```diff
@@ -25,4 +25,8 @@ public String toString() {
   }
 
   public abstract PubSubClientsFactory getPubSubClientsFactory();
+
+  public abstract String getPubSubClusterName();
+
+  public abstract String getRegionName();
```

Comment:
As we treat  `regionName` as one property of `PubSubBrokerWrapper` and we want to Client Adapter Factories to get this information when we create adapters. Do we plan to treat this as one field of properties passed into the client adapter factory `create` interface or you have other ideas?

Reply:
> Do we plan to treat this as one field of properties passed into the client adapter factory create interface or you have other ideas?

I think we should pass it via `VeniceProperties` option that we have.

Reply:
So do you plan to include add the region name into the `VeniceProperties`  in this change?

Reply:
No, it is not going to be part of this change. This change is to unblock integration testing.